### ISSUE TITLE
NVDA does not read embedded text in tab panel

### ISSUE THREAD

**Issue by TechThomas** [nvaccess/nvda#8332]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 

1. Open Firefox 60 on Windows 10
2. Browse to [Tableau Accessibility Demo](https://equalentry.com/tableau-accessibility-demo/)
3. Navigate to tab item **SAT Performance of Admitted Students** 4. NVDA reads application, tab control, SAT Performance of Admitted Students  tab  selected  1 of 3
4. Browse or explore the accessibility elements inside the tab panel

### Expected behavior:
NVDA should be able to read embedded text elements inside of the tab panel. Part of the text that should be read reads **The SAT Performance of Admitted Students dashboard uses two graphs to show SAT score data for students admitted to a hypothetical university.** 

### Actual behavior:
NVDA goes to next interactive element which is **radio button 2008**.

### System configuration:
NVDA version: 2018.1.1
NVDA Installed or portable: Installed

### Other questions:
Have you tried any other versions of NVDA?
No, but VoiceOver, JAWS, and Narrator are all able to read this text.

**Comment by michaelDCurran** [comment]
Can you explain why  this uses a role of application?  Since this example contains text that should be read, and some basic controls, I think staying in browse mode is probably sufficient here.  Right now the user can force this by pressing NVDA+space when within the application.  
have you tried an aria-describedBy on the application pointing to the main text?

**Comment by TechThomas** [comment]
Michael I will discuss and see if aria-describedby will be sufficient. One of the reasons role="application" was used was to ensure that a user that needs the data table alternative for the viz can get keyboard access to it.

**Comment by Adriani90** [comment]
@TechThomas are you still seeing this issue in NVDA 2019.1.1? For me it is not exactly clear how navigation is expected here. When I am exploring the dialog with up and down arrow keys in NVDA browse mode (nvda+ space bar), the text you expect to be read is properly reported.
When exploring the dialog in focus mode with tab and shift+tab (again nvda+ space bar), NVDA reports the interactive elements which can be focused only. Text is not an interactive element, so it is not included in focus mode navigation. Please correct me if I have totally missunderstood the issue, but this example actually works as expected for me.

**Comment by Adriani90** [comment]
Note that if you cahnge the tab to percentile for example, the text about the graph is read properly because it's included in the alt text.

I am closing this issue as works for me.But feel free to comment if I have missunderstood something here.
